Package.md

```diff
@@ -0,0 +1,118 @@
+# Ballerina Twitter Connector
+Connects to Twitter using Ballerina.
+
+# Introduction
+The Twitter API can be used to programmatically retrieve and analyze data, as well as engage with the conversation on Twitter.
+This connector provides access to a variety of different resources in Twitter including the following:
+
+    * Tweets
+    * Users
+ 
+The Twitter API currently consists of two supported versions, as well as different access tiers.
+When user wants to access the connector, they are required to register an application. By default, applications can only access public information on Twitter. Certain operations, such as those responsible for sending or receiving Direct Messages and posting a Tweet, require additional permissions from user before access user information. These permissions are not granted by default; you choose on a per-application basis whether to provide this access, and can control all the applications authorized on your account.
```

Comment:
When user wants to access the connector --> access Twitter through Ballerina Twitter Connector?

Reply:
Ack. Fixed